src/navigation/breadcrumbs/breadcrumbs.jsx

```diff
@@ -221,12 +265,13 @@ function Breadcrumbs(props) {
                             title,
                         } = breadcrumb;
 
-                        const shouldTitleBeEllipsed = title.length > BREADCRUMB_TITLE_MAX_LENGTH;
+                        const shouldTitleBeEllipsed = title.length > titlesMaxLength;
                         const parsedTitle = shouldTitleBeEllipsed ?
```

Comment:
Maybe `shouldTitleBeTruncated` is a better name for `shouldTitleBeEllipsed`.

Reply:
done.

Reply:
Thanks.  Resolving.